tests/test_ocs_basic_install.py

```diff
@@ -91,9 +96,10 @@ def run(**kwargs):
     )
 
     # Test cluster access
-    log.info("Testing access to cluster")
-    os.environ['KUBECONFIG'] = f"{cluster_path}/auth/kubeconfig"
-    run_cmd("oc cluster-info")
+    if not OCP.set_kubeconfig(
```

Comment:
Can we check this at the begining of the test and we will be able to skip quite a bit of code if we are using existing cluster based on cli?

Reply:
Absolutely there is no need for that,  so there are 2 ways to handle this,  (a) either skip this in runner and set the kubecontext (b) do it at the start of the test and the email results will report this test as pass,  We can take the approach (b) for simplicity.